combat-trainer.lic

```diff
@@ -428,9 +428,11 @@ class LootProcess
       if @necro_heal && !game_state.necro_casting?
         game_state.wounds = check_health
         echo "Severity to Wounds: #{game_state.wounds}" if $debug_mode_ct
-        unless game_state.wounds.empty?
-          do_necro_ritual(mob_noun, 'consume', game_state)
-          return false
+        if @heal_threshold <= game_state.wounds.keys.max
```

Comment:
Does this work if you have no wounds?

My understanding is that `game_state.wounds.keys.max` would be `nil` in that case. Comparing `1 <= nil` should result in an exception.

Reply:
very true, and I missed it in testing. I will switch the logic around to check for this.